smac/runhistory/runhistory.py

```diff
@@ -143,8 +166,8 @@ class RunHistory(object):
     """
 
     def __init__(
-            self,
-            overwrite_existing_runs: bool = False,
+        self,
+        overwrite_existing_runs: bool = True,
```

Comment:
why do we overwrite existing runs by default? was does this do/what is the effect?

Reply:
new run for same metadata not updated, but will be now